Iniciando montagem da lista com nomes dos arquivos de áudio e transcrições
[Parallel(n_jobs=4)]: Using backend MultiprocessingBackend with 4 concurrent workers.
[Parallel(n_jobs=4)]: Done  12 tasks      | elapsed:    0.0s
[Parallel(n_jobs=4)]: Done 2040 tasks      | elapsed:    0.7s
[Parallel(n_jobs=4)]: Done 4316 out of 4316 | elapsed:    1.3s finished
Tempo de processamento da vetorizacao 0.23783800000000022
[array([], dtype='<U16')]
Iniciando atualização de caminhos para arquivos de áudio
Iniciando conversão dos audios em espectogramas e log_energy
[Parallel(n_jobs=4)]: Using backend MultiprocessingBackend with 4 concurrent workers.
[Parallel(n_jobs=4)]: Done  12 tasks      | elapsed:    0.4s
[Parallel(n_jobs=4)]: Done 120 tasks      | elapsed:    2.2s
[Parallel(n_jobs=4)]: Done 300 tasks      | elapsed:    5.6s
[Parallel(n_jobs=4)]: Done 552 tasks      | elapsed:   11.0s
[Parallel(n_jobs=4)]: Done 876 tasks      | elapsed:   16.6s
[Parallel(n_jobs=4)]: Done 1272 tasks      | elapsed:   23.6s
[Parallel(n_jobs=4)]: Done 1740 tasks      | elapsed:   31.7s
[Parallel(n_jobs=4)]: Done 2280 tasks      | elapsed:   43.1s
[Parallel(n_jobs=4)]: Done 2960 tasks      | elapsed:   53.2s
[Parallel(n_jobs=4)]: Done 4309 out of 4316 | elapsed:  1.3min remaining:    0.1s
[Parallel(n_jobs=4)]: Done 4316 out of 4316 | elapsed:  1.3min finished
<treinamento.audio.Audio object at 0x7ff8cc4cd470>
[Parallel(n_jobs=4)]: Using backend MultiprocessingBackend with 4 concurrent workers.
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/usuario/mestrado/yorubaSpeechRecognition/environmentYorubaSpeechRecognition/lib/python3.6/site-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/home/usuario/mestrado/yorubaSpeechRecognition/environmentYorubaSpeechRecognition/lib/python3.6/site-packages/joblib/parallel.py", line 253, in __call__
    for func, args, kwargs in self.items]
  File "/home/usuario/mestrado/yorubaSpeechRecognition/environmentYorubaSpeechRecognition/lib/python3.6/site-packages/joblib/parallel.py", line 253, in <listcomp>
    for func, args, kwargs in self.items]
  File "/home/usuario/mestrado/yorubaSpeechRecognition/treinamento/preprocessamento.py", line 431, in gravarDados
    with open(file_audio) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/usuario/mestrado/yorubaSpeechRecognition/treinamento/dadosVetorizados/audios_vetorizados/__wo inu ọdun tuntun pelu__.npy'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/usuario/mestrado/yorubaSpeechRecognition/treinamento/preprocessamento.py", line 446, in <module>
    parallel(delayed(PreProcessamento().gravarDados)(audio) for audio in PreProcessamento().listaGlobalAudios)
  File "/home/usuario/mestrado/yorubaSpeechRecognition/environmentYorubaSpeechRecognition/lib/python3.6/site-packages/joblib/parallel.py", line 1042, in __call__
    self.retrieve()
  File "/home/usuario/mestrado/yorubaSpeechRecognition/environmentYorubaSpeechRecognition/lib/python3.6/site-packages/joblib/parallel.py", line 921, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 644, in get
    raise self._value
FileNotFoundError: [Errno 2] No such file or directory: '/home/usuario/mestrado/yorubaSpeechRecognition/treinamento/dadosVetorizados/audios_vetorizados/__wo inu ọdun tuntun pelu__.npy'
